ChopChop/Models/Ingredients/IngredientCategory.swift

```diff
@@ -0,0 +1,59 @@
+/**
+ Represents a collection of ingredients grouped under a category..
+ */
+class IngredientCategory {
+    var id: Int64?
+    private(set) var name: String
+    private(set) var ingredients: [Ingredient]
+
+    init(name: String, id: Int64? = nil, ingredients: [Ingredient] = []) {
+        self.id = id
+        self.name = name
+
+        ingredients.forEach { $0.ingredientCategoryId = id }
```

Comment:
it will automatically convert all ingredients into same category?

Reply:
Yes, an invariant of an `IngredientCategory` is that all ingredients inside have the same `ingredientCategoryId` as the `id` of the category.